po/ru.po

```diff
@@ -0,0 +1,169 @@
+# Russian translation for spot package.
+# Copyright (C) 2021 THE spot'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the spot package.
+# Automatically generated, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: spot\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-16 19:27+0100\n"
+"PO-Revision-Date: 2021-03-16 19:21+0100\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: Paul Bragin <zeusmods@protonmail.com>\n"
+"Language: ru\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
```

Comment:
Shouldn't this be UTF-8?
See https://raw.githubusercontent.com/GNOME/gimp/master/po/ru.po for an example of a Russian po file

Reply:
I didn't really looked into the docs, so thx for fixing the stuff.